app/javascript/components/user/reducer.js

```diff
@@ -16,6 +16,8 @@ export default (state = DEFAULT_STATE, { type, payload }) => {
       return state.set("isAuthenticated", true).set("id", payload.id).set("username", payload.username);
     case Actions.LOGOUT_SUCCESS:
       return DEFAULT_STATE;
+    case Actions.FETCH_RECORD_TYPES_SUCCESS:
+      return state.set("recordTypes", payload.data.associated_record_types);
```

Comment:
We should also handle the other states including the failure state. For the failure state, we can set the recordTypes to a default value of ["case"]. Can we also update the key name for recordTypes to records? Or if there is a conflict then activeRecords?

Reply:
Done!